tasks/configureApache.yml

```diff
@@ -48,25 +43,16 @@
       - ansible_os_family == 'Debian'
       - use_native_cert
 
-  - name: Change cert source path in Redhat
+  - name: Change cert source path
     set_fact:
-      cert_source_file: '{{ certs_path }}/certs/localhost.crt'
-      key_source_file: '{{ certs_path }}/private/localhost.key'
+      cert_source_file: "{{ certs_path }}/certs/{{ ( ansible_os_family == 'Debian' ) | ternary( 'ssl-cert-snakeoil.pem','localhost.crt' ) }}"
+      key_source_file: "{{ certs_path }}/private/{{ ( ansible_os_family == 'Debian' ) | ternary( 'ssl-cert-snakeoil.key', 'localhost.key' ) }}"
     when:
-      - ansible_os_family == 'RedHat'
-      - use_native_cert
-
-  - name: Change cert source path in Debian
-    set_fact:
-      cert_source_file: '{{ certs_path }}/certs/ssl-cert-snakeoil.pem'
-      key_source_file: '{{ certs_path }}/private/ssl-cert-snakeoil.key'
-    when:
-      - ansible_os_family == 'Debian'
       - use_native_cert
 
   - name: Copy certificate for host
     copy:
-      remote_src: "{{ ( use_native_cert ) | ternary( 'True','False' ) }}"
+      remote_src: "{{ ( use_native_cert ) | ternary( 'true','false' ) }}"
```

Comment:
Why do you need to substitute one boolean value with another?

Reply:
Fixed